circle.yml

```diff
@@ -19,9 +19,17 @@ dependencies:
       source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh &&
       export PATH="$HOME/miniconda/bin:$PATH" &&
       source activate test &&
-      conda install pytest &&
+      conda install pytest entrypoints &&
       conda install pytest-cookies -c conda-forge &&
-      conda install ciocheck -c conda-forge;
+      conda install versioneer -c conda-forge &&
+      conda install ciocheck -c conda-forge &&
```

Comment:
conda install pytest-cookies -c conda-forge &&
conda install ciocheck -c conda-forge
conda install versioneer -c conda-forge &&
conda install ciocheck -c conda-forge &&

Move all these to the spyder-ide channel, I will copy the packages for you